### ISSUE TITLE
OpenStreetMap Copyright and License mention

### ISSUE THREAD

**Issue by BDVGitHub** [FlxSaenger/penpot-map-editor#1]
Hey, 

Thanks for your useful plugin, I see you also use OpenStreetMap data which is great.
Is it possible to add the [Copyright and License](https://www.openstreetmap.org/copyright/en) to the OSM map?
See red circle on the image below.

![image](https://github.com/user-attachments/assets/a6753d9b-d631-4b14-9d04-d004a3e9e918)

[Here ](https://leafletjs.com/) is an example of how to add the [OSM Copyright](https://osmfoundation.org/wiki/Licence/Attribution_Guidelines) and License to a map.
![image](https://github.com/user-attachments/assets/bc6d7ffc-0314-40c6-a7a2-1b18cec9b413)

**Comment by FlxSaenger** [comment]
Hi @BDVGitHub, sorry for the late response; apparently Github didn't notify me, but I saw your post over in the Penpot Community. I probably removed the OSM license link when extending the plugin for usage with Thunderforest and forgot to re-add the initial link, what a pity.

I'll fix this issue tonight when I have access to my test system.

**Comment by BDVGitHub** [comment]
No problem, Thx

**Comment by FlxSaenger** [comment]
Thanks for reaching out. I have added the attribution links for OSM and Thunderforest!

**Comment by BDVGitHub** [comment]
Thx

Is it also possible to update the image on https://penpot.app/penpothub/plugins/map-editor
And the image used on https://github.com/FlxSaenger/penpot-map-editor/blob/main/README.md
Now its only Leaflet that is mentioned.

**Comment by BDVGitHub** [comment]
I tried the plugin, the screen to select the area shows the correct copyright.
Once you press ‘take picture here’ then the copyrights are not on the map you want to use within Penpot.
Is it possible to add these copyrights as well? See the image below:
![afbeelding](https://github.com/user-attachments/assets/4971a216-5ec6-4ec8-a315-fb8c906debee)

**Comment by FlxSaenger** [comment]
Thanks for the comment, I will take a look into it soon!

**Comment by BDVGitHub** [comment]
Any idea when you would have time to fix this issue?

**Comment by FlxSaenger** [comment]
Hi @BDVGitHub the underlying plugin for exporting the map as an image ([leaflet-image](https://github.com/mapbox/leaflet-image)) does not support HTML-like elements such as the contribution-container. A quick workaround that I just tried did not work as expected.

As this issue does not concern any functional problem (e.g., a bug), I have no resources to continue working on it. Correct usage of license credentials is in the responsibility of the end-user. If you can provide any programmatic solution to this problem, feel free to contribute or fork this plugin. Best regards, Felix

**Comment by BDVGitHub** [comment]
@FlxSaenger can you start with giving the right example by updating the image on the Readme file and the image on https://penpot.app/penpothub/plugins/map-editor , since you're the end user u are responsible like you said...

**Comment by FlxSaenger** [comment]
Good idea, I have updated the screenshots and requested Paula from the penpot team to replace the screenshots on their plugin detail page.

**Comment by BDVGitHub** [comment]
@FlxSaenger Thx for updating the screenshots.
May I ask why u use https://github.com/mapbox/leaflet-image instead of for example https://github.com/Leaflet/Leaflet.VectorGrid?
The repo u use is no longer mainted (its 9 years ago something changed) , instead the other repo is mainted...
